### PR TITLE
Time, Location added

### PR DESCRIPTION
**Pull request by MonikaJhingan**
Please review

### CODE REVIEW

**Review by MaruthiPothuganti — COMMENTED** [review]
Codes looks Clean. PR looks good to merge.

**Review by sushil-mod — APPROVED** [review]
short and clean code good to merge

File: src/components/Weather.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        const { data } = await axios.get(URL);
+        const { name, main, weather } = data;
+        setLocation(name);
+        setTemp(Math.round(main.temp));
+        setImage(weather[0].icon);
+      } catch (error) {
+        console.log(error);
+      }
+    })();
+        // eslint-disable-next-line
```
Comment (rahulyadav139) [inline src/components/Weather.jsx:30]: suggestion: we shouldn't manually disable eslint. Instead of it we can use react memoization.

**Review by rahulyadav139 — APPROVED** [review]

File: src/components/Weather.jsx (hunk 1 above)
Comment (MonikaJhingan) [reply src/components/Weather.jsx:30]: ok

**Pull request merged**